src/runtime/runtime_nintendoswitch.go

```diff
@@ -0,0 +1,72 @@
+// +build nintendoswitch
+
+package runtime
+
+type timeUnit int64
+
+const asyncScheduler = false
+
+func postinit() {}
+
+// Entry point for Go. Initialize all packages and call main.main().
+//export main
+func main() int {
+	preinit()
+	run()
+
+	return exit(0) // Call libc_exit to cleanup libnx
+}
+
+// sleepTicks
+func sleepTicks(d timeUnit) {
+	usleep(uint(ticksToNanoseconds(d) / 1000))
```

Comment:
We seem to be dividing by 1000 and then re-multiplying, effectively rounding this down. Is this intentional? Can we flatten this instead to a call to `sleepThread`?

Reply:
Hmm, true. Let me do it.